app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,67 @@
+@import "application";
+.my-header {
+  background-color: $my-black;
+  height: 3.5em;
+  position: fixed;
+  width: 100%;
+}
+
+.header-menu {
+  color: $my-white;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 0.9em;
+  font-weight: 400;
+
+  ul li {
+    display: inline;
+    padding-left: 1em;
+    padding-right: 2em;
+    color: $my-white;
+    text-decoration: none;
+
+    &:hover {
+      color: $my-orange;
+    }
+  }
+
+}
+
+.main-bg {
+  color: $my-black;
+  width: 100%;
+  height: 50.5em;
+  background: image-url('bg6.jpg');
+  background-size: 100%;
+}
+
+.my-container {
+  background-color: $my-brown;
+  border: 2px;
+  border-radius: 10px;
+  padding: 1em;
```

Comment:
Properties should be ordered background-color, border, border-radius, font-family, opacity, padding, position, top